package.json

```diff
@@ -134,6 +135,7 @@
   },
   "devDependencies": {
     "cheerio": "^1.0.0-rc.2",
+    "colors": "^1.1.2",
```

Comment:
You don't need another package to handle colours. We already have [gulp-util that can do colours](https://www.npmjs.com/package/gulp-util#colors).